tests/bluetooth/mesh/brg/src/main.c

```diff
@@ -0,0 +1,430 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/ztest.h>
+#include <zephyr/net/buf.h>
+#include <zephyr/bluetooth/mesh.h>
+#include <stdlib.h>
+
+#include "settings.h"
+#include "brg_cfg.h"
+
+#define TEST_VECT_SZ (CONFIG_BT_MESH_BRG_TABLE_ITEMS_MAX + 1)
+
+static struct test_brg_cfg_row {
+	uint8_t direction;
+	uint16_t net_idx1;
+	uint16_t net_idx2;
+	uint16_t addr1;
+	uint16_t addr2;
+} test_vector[TEST_VECT_SZ];
```

Comment:
I'm pretty sure type from mesh header should be used instead.

Reply:
I want to create test vectors with invalid entries. Like, Netkey Index of 4096. Hence, common type for test purpose is created.